README.md

```diff
@@ -57,12 +59,13 @@ You'll need to mount a directory on the host machine to the image's volume speci
 
 The subdirectory `worlds` can be empty or not exist at all.
 
-There are 4 environment parameters to customize the server's runtime behavior.  2 of these are required to be set, otherwise the container will exit immediately.
+There are 5 environment parameters to customize the server's runtime behavior.  2 of these are required to be set, otherwise the container will exit immediately.
 
 * `VALHEIM_SERVER_NAME`: sets the server's name (**required**, truncated at first whitespace).
 * `VALHEIM_WORLD_NAME`: sets the world's name (**required**, truncated at first whitespace).
 * `VALHEIM_PASSWORD`: sets the server's password.
 * `VALHEIM_PORT`: sets the server's port (default is `2456`).  Recommended not to change this.
+* `VALHEIM_PUBLIC`: Specify, if the server should be listed ingame in Valheim's server list. (0 means it will not be listed; 1 means it will be listed)
```

Comment:
Awesome, thanks for documenting this!  If someone uses `0` for the `public` parameter, how can they join the server, e.g. only through IP?

Reply:
Yes, exactly.
Then you can join, only if you no the address.
I think domains also work, but I didn't try that on my server.
I only tried localhost, but that may still be handled differently from using a URL.